test/built-ins/Function/prototype/bind/get-fn-realm-recursive.js

```diff
@@ -3,23 +3,51 @@
 /*---
 esid: sec-getfunctionrealm
 description: >
-    The realm of a bound function exotic object is the realm of its target
-    function. GetFunctionRealm is called recursively.
+  The realm of a bound function exotic object is the realm of its target function.
+  GetFunctionRealm is called recursively.
 info: |
-    7.3.22 GetFunctionRealm ( obj )
-
-    [...]
-    2. If obj has a [[Realm]] internal slot, then
-       a. Return obj.[[Realm]].
-    3. If obj is a Bound Function exotic object, then
-       a. Let target be obj.[[BoundTargetFunction]].
-       b. Return ? GetFunctionRealm(target).
-features: [cross-realm]
+  Object ( [ value ] )
+
+  1. If NewTarget is neither undefined nor the active function, then
+    a. Return ? OrdinaryCreateFromConstructor(NewTarget, "%Object.prototype%").
+
+  OrdinaryCreateFromConstructor ( constructor, intrinsicDefaultProto [ , internalSlotsList ] )
+
+  [...]
+  2. Let proto be ? GetPrototypeFromConstructor(constructor, intrinsicDefaultProto).
+  3. Return OrdinaryObjectCreate(proto, internalSlotsList).
+
+  GetPrototypeFromConstructor ( constructor, intrinsicDefaultProto )
+
+  [...]
+  3. Let proto be ? Get(constructor, "prototype").
+  4. If Type(proto) is not Object, then
+    a. Let realm be ? GetFunctionRealm(constructor).
+    b. Set proto to realm's intrinsic object named intrinsicDefaultProto.
+  5. Return proto.
+
+  GetFunctionRealm ( obj )
+
+  [...]
+  2. If obj has a [[Realm]] internal slot, then
+    a. Return obj.[[Realm]].
+  3. If obj is a bound function exotic object, then
+    a. Let target be obj.[[BoundTargetFunction]].
+    b. Return ? GetFunctionRealm(target).
+features: [cross-realm, Reflect]
 ---*/
 
-var other = $262.createRealm().global;
-var C = new other.Function();
-C.prototype = null;
-var B = C.bind().bind();
+var realm1 = $262.createRealm().global;
```

Comment:
I am using current Realm as less as possible to avoid false-positive test results.